go-controller/pkg/ovn/ovn.go

```diff
@@ -525,6 +536,16 @@ func (oc *Controller) WatchNodes() error {
 		UpdateFunc: func(old, new interface{}) {
 			oldNode := old.(*kapi.Node)
 			node := new.(*kapi.Node)
+
+			shouldUpdate, err := shouldUpdate(node, oldNode)
+			if err != nil {
+				logrus.Errorf("%v", err)
```

Comment:
@JacobTanenbaum idom here is usually `logrus.Errorf(err.Error())`

Reply:
fixed